app/controllers/buyers/account_plans_controller.rb

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 
 class Buyers::AccountPlansController < Api::PlansBaseController
-  before_action :authorize_manage_account_plans!, only: %i[new create]
-  before_action :authorize_read_account_plans!
+  before_action :authorize_manage_account_plans, only: %i[new create] # rubocop:disable Rails/LexicallyScopedActionFilter #new defined in base controller
```

Comment:
But not defining `new` seems to be the sweet spot, otherwise we get a reek:


![Screenshot 2023-01-31 at 09 00 04](https://user-images.githubusercontent.com/11672286/215700799-de979c7b-8411-4559-86d9-ffafc9b371c8.png)


Like with create, update and destroy. Which warning do we prefer to disable?

Reply:
In the end I removed the suppression comment.